src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec

```diff
@@ -0,0 +1,62 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This .nuspec file is authored based on the file I extracted from Cognitive Services package in nuget.org